.gitignore

```
@@ -42,6 +43,8 @@
 !/[A-Z]*[A-Z]-[A-Z]*[A-Z]/extra/**/*.svg
 !/[A-Z]*[A-Z]-[A-Z]*[A-Z]/extra/**/*.png
 !/[A-Z]*[A-Z]-[A-Z]*[A-Z]/extra/**/*.jpg
+!/[A-Z]*[A-Z]-[A-Z]*[A-Z]/extra/**/*.asm
+!/[A-Z]*[A-Z]-[A-Z]*[A-Z]/extra/popcnt_32bit/**
```

```suggestion
!/[A-Z]*[A-Z]-[A-Z]*[A-Z]/extra/**
```
`extra` wird komplett freigegeben für beliebige Dateiformate und Verzeichnisstrukturen

@ZeroPointMax war korrekt, wie du geschrieben hast, die Zeilen darüber sollten nur auch noch weg